connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/DedicatedMirrorIntegrationTest.java

```diff
@@ -260,9 +261,8 @@ public void testMultiNodeCluster() throws Exception {
             awaitConnectorTasksStart(mirrorMakers.get("node 0"), MirrorHeartbeatConnector.class, sourceAndTarget);
 
             // Create one topic per Kafka cluster per MirrorMaker node
-            final int topicsPerCluster = numNodes;
```

Comment:
With cosmetic changes like this, please watch out so that comments don't become stale. The `// Create one topic per Kafka cluster...` comment should be moved to just above the for-loop since it has nothing to do with the `messagesPerTopic` declaration.